### DISCUSSION TITLE
Upload file in Cefsharp

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [cefsharp/CefSharp#4249] [General]
How Can i upload file in CefSharp!?
in whatsapp i've used IDialogHandler That triggered and fired OpenFIleDialog but file is not shown in whatsapp page...
can i produce this if need...

**Comment by amaitland** [comment]
In future please use discussions to ask questions, don't use the feature request template.

https://github.com/cefsharp/CefSharp/discussions

> in whatsapp i've used IDialogHandler That triggered and fired OpenFIleDialog but file is not shown in whatsapp page...

Did you test with a simpler demo example url to confirm your code is correct? What does your code look like? (No images)

Make sure you are using the latest stable version.

Also please take the time to read over https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/ for some guidance on how to ask for help.
